### ISSUE TITLE
Add VLC volume control

### ISSUE THREAD

**Issue by insin** [insin/vlc_remote#5]
> set volume level to <val> (can be absolute integer, percent or +/- relative value):

```
 ?command=volume&val=<val>
 Allowed values are of the form:
   +<int>, -<int>, <int> or <int>%
```

**Comment by ThisIsBis** [comment]
Added as part of https://github.com/ThisIsBis/vlc_remote/commit/cd588fb7dc36635ce9e8c676cb48b0c39e845a86

**Comment by ThisIsBis** [comment]
Added as part of https://github.com/ThisIsBis/vlc_remote/commit/cd588fb7dc36635ce9e8c676cb48b0c39e845a86

![image](https://user-images.githubusercontent.com/7106415/71555719-25b32080-29e4-11ea-99ff-fc4013f70a90.png)

**Comment by Teezton** [comment]
> Added as part of [ThisIsBis@cd588fb](https://github.com/ThisIsBis/vlc_remote/commit/cd588fb7dc36635ce9e8c676cb48b0c39e845a86)
> 
> ![image](https://user-images.githubusercontent.com/7106415/71555719-25b32080-29e4-11ea-99ff-fc4013f70a90.png)

How to get this version with volume, and it's possible to be a button instead of a slide?

**Comment by ThisIsBis** [comment]
> How to get this version with volume, and it's possible to be a button instead of a slide?

The availability of this update is pending pull-request https://github.com/insin/vlc_remote/pull/10.
The volume can be adjusted with the volume-up/down icon-buttons in addition to the slider - or was there any other UI element you were suggesting?

**Comment by insin** [comment]
Implemented in #10
